dynatrace-operator-google-marketplace/schema.yaml

```diff
@@ -139,6 +139,83 @@ properties:
     title: "Automatically update OneAgent"
     default: true
 
+  codeModules.enabled:
+    type: boolean
+    description: "Enable code modules"
+    title: "Enable code modules"
+  codeModules.volume:
+    type: string
+    description: "Define a volume the OneAgent binary will be taken from"
```

Comment:
We should probably mention that as of now there are 2 possibilities:
1. Providing nothing -> CSI driver will be used
2. Providing emptyDir -> the initContainer + wget approach will be used
3. Providing any other volume -> we are not able to handle this yet

Reply:
expanded description